K_Scaffold/Template v2/scaffold/tabs/_tabs.pug

```diff
@@ -69,8 +93,11 @@ mixin tabs(name="tabs",defaultActiveTabIndex=0)
     //- Navigation header with action button to switch tabs
     nav(class=`tabs__header tabs--${name}__header`)
       each tab, index in tabs
-        +action(tab.button)(data-container-tab=name data-tab=tab.name data-i18n=`tabs-${name}-${tab.name}`)
-          != tab.name
+        if !tab.content
+          - tab.button['data-i18n'] = `tabs-${name}-${tab.name}`;
```

Comment:
The content is most likely a default display when using `data-i18n`. Since having translatable sheet is better because others can contribute for non-english languages, I would systematically add the `data-i18n` attribute.

That way we move a bit toward k-scaffold doing internationalization out of the box (perhaps we'll even be able to auto-generate the `translation.json` file one day?)

Reply:
Doing it this way allows for using icon text (with pictos font or material icons), which shouldn't be translated.

Reply:
Oh okay, never though about icon. This way is better then.